README.md

```diff
@@ -10,102 +10,48 @@
 
 ### Platform operations
 
-Commands below will help you set up and work with Ververica Platform.
-
-#### Generating API token
-
-```shell
-scli platform api-token create \
-  --vvp-url "https://vvp.example.com" \
-  --vvp-namespace "default" \
-  --name "cicd" \
-  --role "editor" \
-  --save-to-kubernetes-secret "vvp/secret"
+Commands below will help you create Flink job (Python or SQL) on Ververica Platform or using Flink Kubernetes Operator.
+
+#### Environments
+
+You can set up your own environment, which will allow you to connect to Ververica/K8S operator. You need to fill
+properties within `config/<env_name>` directory. `config/flink_deployment.yml` is default deployment descriptor file.
```

Comment:
Describe it further what does it mean default here? Default for each environment? Should the user create it himself?

Can we define other files in the same manner and all will be templated?